HW2/P3/mandelbrot.pyx

```diff
@@ -4,19 +4,57 @@ cimport cython
 import numpy
 cimport AVX
 from cython.parallel import prange
-
+cdef: 
+    int NUM_THREADS = 4
 
 cdef np.float64_t magnitude_squared(np.complex64_t z) nogil:
     return z.real * z.real + z.imag * z.imag
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
```

Comment:
I found some other compiler directives that could speed up my program such as @cython.initializedcheck(False)
"If set to True, Cython checks that a memoryview is initialized whenever its elements are accessed or assigned to. Setting this to False disables these checks. Default is True." and @cython.nonecheck(False)

You can find other directives here: http://docs.cython.org/src/reference/compilation.html#compiler-directives